### DISCUSSION TITLE
Is there any examples with nuxt? I can't get it to work

### DISCUSSION THREAD

**Discussion by ReaperMaga** [quarkiverse/quarkus-quinoa#803] [Q&A] [answered]
The portforwarding does not work, when I try to access a nuxt page it just throws me the 404 default quarkus error page.

https://github.com/ReaperMaga/quarkus-nuxt

**Comment by melloware** [comment]
Is Nuxt a server side rendered SSR technology?

**Reply by melloware** [reply]
Also did you try with just a Hello world nuxt app?

**Reply by ReaperMaga** [reply]
It is ssr yes, I will try later on with a Hello world nuxt app, but it should be same except the tailwindcss and some eslint config

**Reply by melloware** [reply]
Ok I can test it tomorrow as well. I don't think anyone has ever reported trying Nuxt

**Reply by ReaperMaga** [reply]
Thank you for the fast answer

**Reply by melloware** [reply]
@ReaperMaga I created this simple example if you run `mvn quarkus:dev` it seems to work fine with Nuxt...
[quinoa-nuxt.zip](https://github.com/user-attachments/files/17702546/quinoa-nuxt.zip)

**Reply by melloware** [reply, marked as answer]
Confirmed Nuxt won't work because when you package for production is an SSR app ` You can preview this build using node .output/server/index.mjs` but Quinoa is the HTTP server not Node.js so this will never work with Quinoa.

**Reply by ReaperMaga** [reply]
Thats sad, still thanks for helping me!
